Reporting-ETL/KK/Reporting_ETL_Job/items/final_etl/process/Operator_dimension_0.1.item

```diff
@@ -113,33 +106,34 @@
-    <elementParameter field="COMPONENT_LIST" name="CONNECTION" value="" show="false"/>
+    <elementParameter field="COMPONENT_LIST" name="CONNECTION" value=""/>
     <elementParameter field="TEXT" name="HOST" value="context.report_server"/>
     <elementParameter field="TEXT" name="PORT" value="context.report_port"/>
     <elementParameter field="TEXT" name="DBNAME" value="context.report_database"/>
-    <elementParameter field="TEXT" name="TYPE" value="MYSQL" show="false"/>
+    <elementParameter field="TEXT" name="TYPE" value="MYSQL"/>
     <elementParameter field="TEXT" name="USER" value="context.report_login"/>
     <elementParameter field="PASSWORD" name="PASS" value="8pW6Nmy2kudEloY+yZexUg0QoRFSbC0m"/>
     <elementParameter field="DBTABLE" name="TABLE" value="&quot;operator_dimension&quot;"/>
     <elementParameter field="QUERYSTORE_TYPE" name="QUERYSTORE" value="&quot;&quot;"/>
-    <elementParameter field="TECHNICAL" name="QUERYSTORE:REPOSITORY_QUERYSTORE_TYPE" value="" show="false"/>
+    <elementParameter field="TECHNICAL" name="QUERYSTORE:REPOSITORY_QUERYSTORE_TYPE" value=""/>
     <elementParameter field="TECHNICAL" name="QUERYSTORE:QUERYSTORE_TYPE" value="BUILT_IN"/>
-    <elementParameter field="MEMO_SQL" name="QUERY" value="&quot;INSERT INTO `operator_dimension` (`operator_code`,`Operator`,`DW_LastUpdatedDtim`) &#xA;VALUES ('D','Aircel',now()),&#x9; &#xA;('A','Airtel',now()),&#x9;&#x9; &#xA;('B','BSNL',now()),&#x9; &#xA;('L','BPL',now()),&#xA;('C','Datacom Solutions',now()),&#x9; &#xA;('H','HFCL Infotel',now()),&#xA;('I','Idea',now()),&#x9; &#xA;('M','MTNL',now()),&#x9; &#xA;('R','Reliance GSM',now()),&#x9; &#xA;('E','Reliance CDMA',now()),&#x9; &#xA;('S','S. Tel',now()),&#xA;('Y','Shyam Telecom',now()),&#x9; &#xA;('P','Spice',now()),&#xA;('W','Swan Telecom',now()),&#xA;('T','Tata Docomo',now()),&#x9; &#xA;('U','Uninor',now()),&#x9; &#xA;('V','Vodafone',now()),&#x9; &#xA;('9','Unknown',now())&#xA;ON DUPLICATE KEY UPDATE    &#xA;Operator=VALUES(Operator), &#xA;DW_LastUpdatedDtim=VALUES(DW_LastUpdatedDtim);&#xA;&quot;"/>
+    <elementParameter field="MEMO_SQL" name="QUERY" value="&quot;INSERT INTO `ananya_kilkari`.`operator_dimension` (`operator_code`,`Operator`,`DW_LastUpdatedDtim`) &#xA;VALUES ('D','Aircel',now()),&#x9; &#xA;('A','Airtel',now()),&#x9;&#x9; &#xA;('B','BSNL',now()),&#x9; &#xA;('L','BPL',now()),&#xA;('C','Datacom Solutions',now()),&#x9; &#xA;('H','HFCL Infotel',now()),&#xA;('I','Idea',now()),&#x9; &#xA;('M','MTNL',now()),&#x9; &#xA;('R','Reliance GSM',now()),&#x9; &#xA;('E','Reliance CDMA',now()),&#x9; &#xA;('S','S. Tel',now()),&#xA;('Y','Shyam Telecom',now()),&#x9; &#xA;('P','Spice',now()),&#xA;('W','Swan Telecom',now()),&#xA;('T','Tata Docomo',now()),&#x9; &#xA;('U','Uninor',now()),&#x9; &#xA;('V','Vodafone',now()),&#x9; &#xA;('9','Unknown',now())&#xA;ON DUPLICATE KEY UPDATE    &#xA;Operator=VALUES(Operator), &#xA;DW_LastUpdatedDtim=VALUES(DW_LastUpdatedDtim);&#xA;&quot;"/>
```

Comment:
This looks wrong to me.
Why is a target database ('ananya_kilkari') added here?
I remember taking out this before, might it be that you started with an old version?